src/import/loadConfigurationFromIgResource.ts

```diff
@@ -64,7 +64,8 @@ export function loadConfigurationFromIgResource(igRoot: string): Configuration |
       version: igResource.version,
       fhirVersion: igResource.fhirVersion ?? [],
       dependencies: igResource.dependsOn?.filter(dep => dep.packageId && dep.version),
-      FSHOnly: true
+      FSHOnly: true,
+      parameters: igResource.definition?.parameter ?? []
```

Comment:
Another petty comment, but... I like keeping `FSHOnly` as the last thing.  This would also mirror the order that we document it on FSHSchool ([link](https://fshschool.org/docs/sushi/configuration/)).

Reply:
Addressed in commit a080a34.